src/ServiceControl.Audit.Persistence.RavenDB/Extensions/RavenQueryExtensions.cs

```diff
@@ -19,6 +20,45 @@ static class RavenQueryExtensions
             return source;
         }
 
+        public static IQueryable<MessagesViewIndex.SortAndFilterOptions> FilterBySentTimeRange(this IQueryable<MessagesViewIndex.SortAndFilterOptions> source, string range)
+        {
+            if (string.IsNullOrWhiteSpace(range))
+            {
+                return source;
+            }
+
+            var filters = range.Split(SplitChars, StringSplitOptions.None);
+            DateTime from, to;
+            try
+            {
+
+                if (filters.Length == 2)
+                {
+                    from = DateTime.Parse(filters[0], CultureInfo.InvariantCulture, DateTimeStyles.RoundtripKind);
+                    to = DateTime.Parse(filters[1], CultureInfo.InvariantCulture, DateTimeStyles.RoundtripKind);
+                    source.Where(m => m.TimeSent >= from && m.TimeSent <= to);
```

Comment:
Does the TO not need to be exclusive?
```suggestion
                    source.Where(m => m.TimeSent >= from && m.TimeSent < to);
```

Reply:
no, the range is inclusive, it uses the between syntax of Raven